packages/block-library/src/image/edit.native.js

```diff
@@ -81,7 +81,14 @@ class ImageEdit extends React.Component {
 
 		const { attributes, setAttributes } = this.props;
 
-		if ( attributes.id && ! isURL( attributes.url ) ) {
+    // This will warn when we have `id` defined, while `url` is undefined.
```

Comment:
This whole block of code if shifted left, please can you make sure that it's properly aligned?

Probably the best option to prevent those kinds of commits is to cd into `gutenberg-mobile/gutenberg` and run `npm install` - which will add hooks so after that whenever you want to commit something hooks will trigger lint checks and you won't be able to commit without resolving lint errors.

Reply:
Thank you @marecar3 . I have pushed a fix for the lint errors. I added those lines from the "inner-inner" directory while working / testing with WPAndroid, so I didn't catch that I had spaces instead of tabs :grimacing: . Thanks for reviewing! :)